torchx/schedulers/test/slurm_scheduler_test.py

```diff
@@ -395,6 +395,19 @@ def test_describe_running(self, run: MagicMock) -> None:
         self.assertEqual(out.msg, "RUNNING")
         self.assertEqual(out.state, specs.AppState.RUNNING)
 
+    @patch("subprocess.run")
+    def test_list(self, run: MagicMock) -> None:
```

Comment:
What do you think of adding integration tests as well?

Reply:
Added